server/fleet/activities.go

```diff
@@ -708,3 +710,19 @@ func (a ActivityTypeEditedMacOSMinVersion) Documentation() (activity string, det
   "deadline": "2023-06-01"
 }`
 }
+
+type ActivityTypeReadHostDiskEncryptionKey struct {
+	HostDisplayName string `json:"host_display_name"`
```

Comment:
I wonder if we should add the `HostID` (and generally, always add the ID with the displayable thingy in the activity)? This is so that we enable scriptable things with activities (use the API to list the activities, get the host ID, use the API to do things with that host, things like that)... Thoughts?